src/instructlab/configuration.py

```diff
@@ -389,6 +390,7 @@ class Config(BaseModel):
 def get_default_config() -> Config:
     """Generates default configuration for CLI"""
     return Config(
+        version="1.0.0",
```

Comment:
We're going to want to set this as a variable somewhere else in this file. 

I would put it right above the variable for the config file name itself, https://github.com/instructlab/instructlab/blob/main/src/instructlab/configuration.py#L39.

Reply:
Added a var